debug/update_image.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Mapbox GL JS debug page</title>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+    <link rel='stylesheet' href='/dist/mapbox-gl.css' />
+    <style>
+        body { margin: 0; padding: 0; }
+        html, body, #map { height: 100%; }
+    </style>
+</head>
+
+<body>
+<div id='map'></div>
+
+<script src='/dist/mapbox-gl-dev.js'></script>
+<script src='/debug/access_token_generated.js'></script>
+<script>
+
+// Source: https://radar.weather.gov/ridge/Conus/RadarImg/southeast_radaronly.gfw
+const origin = {
+    lng: -90.24006072802854, 
+    lat: 36.928147474567794
+};
+
+const size = {
+    width: 840,
+    height: 800,
+}
+
+const opposite = {
+    lng: origin.lng + size.width * 0.017971305190311,
+    lat: origin.lat + size.height * -0.017971305190311,
+}
+
+const center = {
+    lng: origin.lng + (opposite.lng - origin.lng) / 2,
+    lat: origin.lat + (opposite.lat - origin.lat) / 2,
+}
+
+const map = new mapboxgl.Map({
+    container: 'map',
+    minZoom: 1,
+    zoom: 6,
+    center: [center.lng, center.lat],
+    style: 'mapbox://styles/mapbox/streets-v10',
+    hash: false
+});
+
+const path = "/test/integration/image/";
+const images = [
```

Comment:
I'm not sure if we need quite so many gifs to be added to the library. You can illustrate this with just a few. In fact, you can probably modify this to use `'https://www.mapbox.com/mapbox-gl-js/assets/radar' + currentImage + '.gif'` as seen [here](https://www.mapbox.com/mapbox-gl-js/example/animate-images/) and eliminate the need for these gifs entirely.